website/content/en/docs/installation/_index.md

```diff
@@ -36,7 +36,7 @@ export OS=$(uname | awk '{print tolower($0)}')
 Download the binary for your platform:
 
 ```sh
-export OPERATOR_SDK_DL_URL=https://github.com/operator-framework/operator-sdk/releases/download/v1.5.1
+export OPERATOR_SDK_DL_URL=https://github.com/operator-framework/operator-sdk/releases/v1.5.2
```

Comment:
`make generate` seemed to change this. Is the change in URL expected ?

Reply:
The script is bugged it seems, probably because it was not updated correctly since the latest -> X.Y.Z link change
```suggestion
export OPERATOR_SDK_DL_URL=https://github.com/operator-framework/operator-sdk/releases/download/v1.5.2
```